<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerrainEditor</class>
 <widget class="QMainWindow" name="TerrainEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1267</width>
    <height>896</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Terrain Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1267</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionClear_Console"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionMap_Options"/>
    <addaction name="actionFlora_Options"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOptions"/>
   <addaction name="menuTools"/>
   <addaction name="menuView"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="constructionLayersTree">
   <property name="minimumSize">
    <size>
     <width>89</width>
     <height>350</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Construction Layers</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeView" name="layerTree">
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="contextProperties">
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="contextPropertiesWidget">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="groups">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>524287</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Groups</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTabWidget" name="groupTabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <widget class="swged::ShaderGroupWidget" name="tabShaderGroup">
        <attribute name="title">
         <string>Shader</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </widget>
       <widget class="QWidget" name="tabFloraGroup">
        <attribute name="title">
         <string>Flora</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabRadialGroup">
        <attribute name="title">
         <string>Radial</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabFractalGroup">
        <attribute name="title">
         <string>Fractal</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabEnvironmentGroup">
        <attribute name="title">
         <string>Environment</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
    <zorder>groupTabWidget</zorder>
    <zorder>groupTabWidget</zorder>
   </widget>
  </widget>
  <widget class="QDockWidget" name="console">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>89</width>
     <height>75</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>100</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionClear_Console">
   <property name="text">
    <string>Clear Console</string>
   </property>
  </action>
  <action name="actionMap_Options">
   <property name="text">
    <string>Map Options</string>
   </property>
  </action>
  <action name="actionFlora_Options">
   <property name="text">
    <string>Flora Options</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swged::ShaderGroupWidget</class>
   <extends>QWidget</extends>
   <header>terrain/shader_group_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
